src/server/system_services/schemas/nsfs_account_schema.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
                 properties: {
                     uid: { type: 'number' },
                     gid: { type: 'number' },
+                    supplemental_groups: { type: 'string' },
```

Comment:
I began from that, but it will cause issues with updating the configuration as it is a list. also we will need to convert on the native side anyway as we will convert between napi array to std::vector. I thought it was better this way as we don't need the list outside the native and it simplifies the process. also it looks better when looking at the configuration